### PR TITLE
Add tasks and vars for installing and configuring fluentd

### PR DESCRIPTION
**Pull request by evansmurithi**

### CODE REVIEW

File: molecule/default/verify.yml [hunk 1]
```diff
@@ -4,6 +4,21 @@
 - name: Verify
```
Comment (jasonrogena) [inline molecule/default/verify.yml]: @evansmurithi do we still need this file?

**Review by jasonrogena — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,22 +1,54 @@
...
+fluentd_system_user: "td-agent"
+# list of groups to add the td-agent user to
+fluentd_monitoring_groups:
+  - adm
+
+# input config
+# directory to store the last read position of a log
+fluentd_source_log_pos_directory: "/var/log/td-agent"
+# log paths to tail
+fluentd_log_paths:
```
Comment (jasonrogena) [inline README.md:32]: @evansmurithi would it be possible to also, by default, configure the following log paths:
1. /var/log/auth.log
2. /var/log/syslog

**Review by jasonrogena — APPROVED** [review]

File: molecule/default/verify.yml (hunk 1 above)
Comment (evansmurithi) [reply molecule/default/verify.yml]: No, forgot to remove it

File: README.md (hunk 2 above)
Comment (evansmurithi) [reply README.md:32]: @jasonrogena yes we can. I've already tested with auth.log, lemme test with syslog

**Review by jasonrogena — APPROVED** [review]

**Pull request merged**